### ISSUE TITLE
HA model in continuous time

### ISSUE THREAD

**Issue by danieljschmidt** [greimel/distributional-macroeconomics#13]
While exploring the continuous-time-new.jl notebook, I noticed a few things that do not make much sense to me:

- In the stationary equilibrium, 20% of the probability mass sits in the first income state, 80% in the second income state, 0% in the high income state. If I add more income states, the distribution looks even weirder (see table in notebook). From what I understand, you choose grid points based on the stationary distribution of the income process (called ```distribution``` in the notebook), therefore all points in the income grid should have a nonzero probability mass. 

- Why is most of the probability mass concentrated at $a=0$ even though there does not seem to be a higher interest rate for negative assets in the model?

I do not understand the continuous-time notebook well enough to see if there is a problem with the code or if I simply misunderstand something.

I have illustrated my questions in the notebook (in a separate branch called cont-time-issue).

**Comment by greimel** [comment]
I agree that this is odd. And I am not sure what to about it. You've just found evidence that numerical work is messy.

Here are a few things that could be checked.

* If one discretizes the income process (as in `continuous-time-processes.jl`), will the stationary distribution look the same?
* If you look at the figure below it seems as if something is going bad for low interest rates. Probably there's a problem with discretization of assets around 0? For big r the distribution looks quite nice.

<img width="675" alt="image" src="https://user-images.githubusercontent.com/6280307/174075554-7adfdcde-cd7f-409e-be85-5e538d6b40e1.png">

But to be honest I don't have any resources to dive into this right now.

**Comment by danieljschmidt** [comment]
Thanks for having a look at the notebook! I understand that you cannot spend more time on this

I will make an attempt at finding the problem during the next days (using your advice as a starting point), but if I do not manage, I will come up with a different idea for the tutorial
